db/seeds.rb

```diff
@@ -1 +1,75 @@
+class Seed
+  def run
+    create_borrowers(10)
+    create_lenders(10)
+    create_loan_requests_for_each_borrower(3)
+    create_categories(7)
+  end
 
+  def lenders
+    User.where(role: 0)
+  end
+
+  def borrowers
+    User.where(role: 1)
+  end
+
+  def create_lenders(quantity)
+    quantity.times do
+      name = Faker::Name.name
+      email = Faker::Internet.email
+      user = User.create(name: name, password: 'password', email: email, role: 0)
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.